pkg/cmd/cli/cmd/rsync/copyrsync.go

```diff
@@ -34,16 +34,15 @@ var rshExcludeFlags = sets.NewString("delete", "strategy", "quiet", "include", "
 
 func newRsyncStrategy(f *clientcmd.Factory, c *cobra.Command, o *RsyncOptions) (copyStrategy, error) {
 	// Determine the rsh command to pass to the local rsync command
-	rsh := cmdutil.SiblingCommand(c, "rsh")
-	rshCmd := []string{rsh}
+	rshCmd := cmdutil.SiblingCommand(c, "rsh")
 	// Append all original flags to rsh command
 	c.Flags().Visit(func(flag *pflag.Flag) {
 		if rshExcludeFlags.Has(flag.Name) {
 			return
 		}
 		rshCmd = append(rshCmd, fmt.Sprintf("--%s=%s", flag.Name, flag.Value.String()))
 	})
-	rshCmdStr := strings.Join(rshCmd, " ")
+	rshCmdStr := strings.Join(cmdutil.EscapeCommand(rshCmd), " ")
```

Comment:
EscapeCommand takes a []string, iterates over all of the elements quoting them if needed, and then returns a []string that then needs to be joined.  Why should it go in the rshCmd = append ... instead of below it?